src/confluent_kafka/admin/_config.py

```diff
@@ -133,6 +164,8 @@ def __init__(self, restype, name,
         else:
             self.set_config_dict = dict()
 
+        self.incremental_configs = list(incremental_configs or [])
```

Comment:
I don't know how correct this is. For example, in string case if `incremental_configs = "abcd"` then `self.incremental_configs = ['a', 'b', 'c', 'd']`. This is anyway incorrect input itself but the error message that the user will get will not be correct.

I would say, change it to `self.incremental_configs = incremental_configs or []`

Reply:
That won't copy the original list. In this case given the string is iterable it's ok following duck typing to treat it as a list an make it fail because of the type.

Reply:
It will fail but the error won't be correct. Changing it to `self.incremental_configs = incremental_configs or []` is incorrect as well.

Reply:
Anyways. Its a corner case which can be solved by reading the doc. So, its fine to keep it this way.

Reply:
I don't think the error is incorrect, you have passed a str that is an iterable of str. The error says that the first element 'a' of the collection is not a ConfigEntry